tests/CommonTestUtils/Epilogue.cs

```diff
@@ -0,0 +1,83 @@
+﻿using System.ComponentModel;
+
+namespace CommonTestUtils
+{
+    /// <summary>
+    /// This class supports TestCleanUpAttribute. See TestCleanUpAttribute.cs
+    /// for technical details.
+    /// </summary>
+    public static class Epilogue
```

Comment:
Rather do not declare it `static` but instantiate it in `EpilogueAttribute`.

Reply:
Hmm, but there are consumers calling methods on it. This would force those consumers to call those methods through `EpilogueAttribute`, which seems like they're now addressing an implementation detail.